Remotion/Web/Development.WebTesting.IntegrationTests.Infrastructure/InternetExplorer/InternetExplorerBrowserFactory.cs

```diff
@@ -21,14 +21,13 @@
 using OpenQA.Selenium.IE;
 using Remotion.Utilities;
 using Remotion.Web.Development.WebTesting.BrowserSession;
-using Remotion.Web.Development.WebTesting.BrowserSession.InternetExplorer;
 using Remotion.Web.Development.WebTesting.Configuration;
-using Remotion.Web.Development.WebTesting.WebDriver.Configuration.InternetExplorer;
-
-namespace Remotion.Web.Development.WebTesting.WebDriver.Factories.InternetExplorer
+using Remotion.Web.Development.WebTesting.WebDriver;
+using Remotion.Web.Development.WebTesting.WebDriver.Factories;
+namespace Remotion.Web.Development.WebTesting.IntegrationTests.Infrastructure.InternetExplorer
 {
   /// <summary>
-  /// Responsible for creating a new InternetExplorer browser, configured based on <see cref="IInternetExplorerConfiguration"/> and <see cref="DriverConfiguration"/>.
+  /// Responsible for creating a new InternetExplorer browser, configured based on <see cref="DriverConfiguration"/> and <see cref="IInternetExplorerConfiguration"/>.

```

Comment:
why the interface?

Reply:
I literally just moved existing IE code, I can remove the interface if you like

Reply:
yes, please. straight-forward optimzations are useful even if we just move code. it reduces mental strain when we come by this code in a year or two.